Makefile

```
@@ -339,14 +341,51 @@ undeploy: kustomize delete-webhook-secret-file ## Undeploy operator and agent fr
 	$(MAKE) undeploy FORCE=--force
 
-.PHONY: deploy-examples
-deploy-examples: ## Deploy the examples to the K8s cluster specified in ~/.kube/config.
-	@echo "Create Namespace based GKMCache"
-	$(KUBECTL) apply -f examples/namespace/RWO/
-	$(KUBECTL) apply -f examples/namespace/ROX/
-	@echo "Create Cluster based ClusterGKMCache"
-	$(KUBECTL) apply -f examples/cluster/RWO/
-	$(KUBECTL) apply -f examples/cluster/ROX/
+.PHONY: gen-apply-example
+gen-apply-example:
```

checkmake (0.3.2)

[warning] Target body for "gen-apply-example" exceeds allowed length of 5 lines (8).

(maxbodylength)